deploy/examples/common.yaml

```
@@ -280,6 +299,7 @@ rules:
       - cephfilesystemmirrors
       - cephfilesystemsubvolumegroups
       - cephblockpoolradosnamespaces
+      - cephcosidriver
```

With this change, I am able to start the operator from your branch without any errors. Does this work for you? 
```suggestion
      - cephcosidrivers
```

---

How long did u ran the operator? Operator won't crashloopoff at least for 10-15 mins, then only the error will pop up in the logs AFAIR.  I ran into same issue even rbac changes in common.yaml

---

Yaa it worked, I missed something with buil earlier